firefly_client/firefly_client.py

```diff
@@ -63,6 +63,9 @@ class FireflyClient(WebSocketClient):
         If True, bring up a Jupyterlab or a browser tab for Firefly. Default False.
     start_browser_tab: `bool`
         If True, and start_tab is true and in use_lab_env is true then start a browser tab. Default False.
+    token: `str` or None
+        A token for connecting to a Firefly server that requires
+        authentication.
```

Comment:
Can you state here in the docstrings what HTTPS header is actually set when a token is provided?

Reply:
(I think it's going to be "Authorization: Bearer" but I think this should just be stated explicitly to aid future users.)

Reply:
I'll add this but I worry this strays from "how to use it" into "how it works".